server/middleware/link/setupRequestLink.ts

```diff
@@ -1,27 +1,13 @@
 import express, { Router } from 'express'
-import session from 'express-session'
 import flash from 'connect-flash'
 import RequestLinkController from '../../routes/link/RequestLinkController'
-import config from '../../config'
 import MagicLinkService from '../../services/link/MagicLinkService'
 
 export default function setUpRequestLink(magicLinkService: MagicLinkService): Router {
   const router = express.Router()
   const requestLinksController = new RequestLinkController(magicLinkService)
 
-  // TODO This uses the same max session age and secret as for a logged in user. Consider separating these into different env vars.
-  router.use(
```

Comment:
Setting up the session here is unnecessary as it is already setup in `setupWebSession.ts`, complete with using the redis store rather than a memory store.